internal/global/uuid.go

```diff
@@ -28,3 +28,12 @@ const ManagedByPipelineTag = "banzaicloud-pipeline-uuid"
 func PipelineUUID() string {
 	return viper.GetString(config.PipelineUUID)
 }
+
+// AlwaysPipelineUUID returns PipelineUUID or "unknown" if none set
+func AlwaysPipelineUUID() string {
```

Comment:
Can't we just set the default in viper to "unknown"?

Reply:
i'd like to make clear that this is not an uuid, but a fallback for cases when a non-empty value is mandatory